src/main/java/beam/utils/BeamCalcLinkStats.java

```diff
@@ -160,35 +160,52 @@ public void writeFile(final String filename) {
                         Link link = this.network.getLinks().get(linkId);
 
                         out.write(linkId.toString());
-                        out.write("," + link.getFromNode().getId().toString());
-                        out.write("," + link.getToNode().getId().toString());
+                        // out.write("," + link.getFromNode().getId().toString());
+                        writeCommaAndStr(out, link.getFromNode().getId().toString());
```

Comment:
What are your thoughts on removing the `out` via an implicit? Maybe hiding too much and would make it harder to debug for others?

Reply:
@JustinPihony , as far as I know there is no `implicits` in Java. Or you want to extract this to Scala method with `implicit` parameter?